### DISCUSSION TITLE
Does the springshell 0day impact DBeaver?

### DISCUSSION THREAD

**Discussion by maxtheking** [dbeaver/dbeaver#16099] [Q&A] [answered]
As the title states, does the Springshell 0day exploit impact DBeaver?
I know the vulnerability applies to JDK 9.0 or later, and DBeaver comes with OpenJDK 11.
Any news on this?

Thanks in advance.

**Comment by flambchap** [comment, marked as answer]
Hi. None of the DBeaver versions use Spring Framework. Therefore, this CVE does not impact DBeaver.
